ui/src/App.vue

```diff
@@ -187,12 +234,8 @@ export default {
   },
 
   mounted() {
-    fetch('/update/identity').then(async (response) => {
```

Comment:
I think you made this to test in the local environment. 
Make sure to revert these lines for deployment on an MCU.

Reply:
We can use `if(process.env.NODE_ENV === 'production')` to skip this HTTP request

Reply:
You're right. I should've excluded that in the PR.

Reply:
Added a commit to correct his.